Modules/@babylonjs/react-native/shared/XrAnchorHelper.h

```diff
@@ -183,3 +183,47 @@ namespace Babylon::Plugins::NativeXr
 }
 #endif
 #endif
+
+#if __has_include("IXrContextARKit.h")
+#include "IXrContextARKit.h"
+#if __has_include("jsi/jsi.h")
+namespace Babylon::Plugins::NativeXr
+{
+    bool TryGetNativeAnchor(facebook::jsi::Runtime& jsiRuntime, facebook::jsi::Value& jsAnchor, ARAnchor*& nativeAnchor)
+    {
+        nativeAnchor = nullptr;
+        uintptr_t nativeAnchorPtr{reinterpret_cast<uintptr_t>(nullptr)};

```

Comment:
nit: could use auto here

Reply:
going to leave as uintptr_t to stay consistent with rest of file/avoid editing the rest of the file.